LocalParks.Infrastructure.Tests/ChartBuilderTests.cs

```diff
@@ -0,0 +1,122 @@
+using LocalParks.Core.Chart;
+using LocalParks.Infrastructure.Builders;
+using Xunit;
+
+namespace LocalParks.Infrastructure.Tests
+{
+    public class ChartBuilderTests
+    {
+        [Fact]
+        public void WHEN_Chart_Is_Constructed_THEN_Chart_Cannot_be_retrieved()
```

Comment:
is this chart builder something you found online?

Reply:
This is something I wrote as I couldn't find any packages for it, designed to produce charts for the ChartJS package 
https://www.chartjs.org/docs/latest/?q=lazy